### PR TITLE
Feat/rework eth cashout

### PR DESCRIPTION
**Pull request by csrapr**

### CODE REVIEW

**Comment by csrapr** [comment]
[Overhaul ETH cash-out behaviour](https://trello.com/c/gf8v3QAa/696-overhaul-eth-cash-out-behaviour)

File: lib/mocks/trader.js [hunk 1]
```diff
@@ -19,7 +20,7 @@ var Trader = function () {
 
   this.txLimit = BN(100)
   this.fiatTxLimit = BN(150)
-  this.zeroConfLimit = 20
+  this.zeroConfLimits = this.buildZeroConfLimits(Object.keys(Tx.coins))
```
Comment (chaotixkilla) [inline lib/mocks/trader.js:23]: lodash is already in use in this file, you can just call `_.keys()`

**Review by chaotixkilla — CHANGES_REQUESTED** [review]
Minimal change

**Review by chaotixkilla — APPROVED** [review]

File: lib/coins/utils.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const coins = {
   ETH: {
     displayScale: 15,
     unitScale: 18,
-    zeroConf: false
+    zeroConf: true
```
Comment (RafaelTaranto) [inline lib/coins/utils.js:11]: Where is thing being used?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/coins/utils.js (hunk 2 above)
Comment (csrapr) [reply lib/coins/utils.js:11]: On line 3400:

`return coin.zeroConf && tx.fiat.gte(this.trader.zeroConfLimits[tx.cryptoCode])`

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**